.github/workflows/create-update-release.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: Update Major Version of Prefect Deploy Action
+on:
+  push:
+    tags:
+      # Match the version format to not catch vx major releases
```

Comment:
what does this mean?

Reply:
Means we don't want to run this action when a major version (i.e v5) is tagged and released

Reply:
what is the `else` statement doing then if not running on a major release?